src/utils.ts

```diff
@@ -191,6 +198,32 @@ export class Utils {
         }
     }
 
+    /**
+     * If repository input was set, extract CLI download details from the first JF_ENV_* environment variable.
+     * @param repository - Remote repository in Artifactory pointing to https://releases.jfrog.io/artifactory/jfrog-cli/
+     * @returns the download details.
+     */
+    public static extractDownloadDetails(repository: string): DownloadDetails {
+        if (repository === '') {
+            return Utils.DEFAULT_DOWNLOAD_DETAILS;
+        }
+        for (let serverToken of Utils.getServerTokens()) {
+            let serverObj: any = JSON.parse(Buffer.from(serverToken, 'base64').toString());
+            if (!serverObj || !serverObj.artifactoryUrl) {
+                continue;
+            }
+            let artifactoryUrl: string = serverObj.artifactoryUrl.replace(/\/$/, '');
+            let results: DownloadDetails = { artifactoryUrl, repository } as DownloadDetails;
+            if (serverObj.user && serverObj.password) {
+                results.auth = 'Basic ' + Buffer.from(serverObj.user + ':' + serverObj.password).toString('base64');
+            } else if (serverObj.accessToken) {
+                results.auth = 'Bearer ' + Buffer.from(serverObj.accessToken).toString('base64');
+            }
+            return results;
+        }
+        throw new Error(`'download-repository' input provided, but no Artifactory server found`);
```

Comment:
I guess this error can be thrown if no secret with a ```JF_ENV_``` prefix was found. I suggest we add this as a hint to the error description. This error description may come in handy for someone who didn't set the scret correctly or forgot to set it.

Reply:
This error should be thrown also, for example when the user exported Xray details without Artifactory URL.